python/helpers/pydev/pydevd_file_utils.py

```diff
@@ -73,10 +73,29 @@
 
 normcase = os_normcase # May be rebound on set_ide_os
 
+def convert_to_long_pathname(filename):
```

Comment:
I think it's not good idea to define function `convert_to_long_pathname` twice. Let's rewrite it using inner functions.